configs/kubernetes-nightly-configmap.yaml

```diff
@@ -1,9 +1,10 @@
+apiVersion: v1
 kind: ConfigMap
 metadata:
   name: publisher-config
 data:
   config: |
-    source-org: kubernetes-nightly
+    source-org: kubernetes
```

Comment:
the source-org would still be kubernetes I believe. Otherwise, import paths do not work.

Reply:
agree